### ISSUE TITLE
OGCS terminates on startup [0x80029C4A]

### ISSUE THREAD

**Issue by Thezzah** [phw198/OutlookGoogleCalendarSync#1214]
> Checklist:-
> * [X] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [X] Reproduce the issue
> * [X] Complete the information below
> * [X] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v2.8.6 Alpha
**Edition**: Installed
**Sync Direction**: O<->G

**Bug Description**  
Application terminates directly on startup and i get the error message: 0x80029C4A (and FAQs COM Errors opens).
I've tried to reboot and done an office repair. I've done all steps under "Search for Invalid Registry Key" and all three things under "Registry Fixes" (Fix the Interface, Fix the TypeLib and Ensure MSOUTL.LIB is Registered)

The only thing i found was that the version in HKEY_CLASSES_ROOT\Interface\{00063001-0000-0000-C000-000000000046}\TypeLib was 9.4 instead of 9.6, but it still doesn't work after I've changed it.

**Steps to Reproduce the Issue**  
1. Open OGCS (or autostart on computer startup)

**Any other information**
Log sent through Google forms
The problem started 1-3 weeks ago without me doing anything intentional with my office/outlook version.
Outlook version: Outlook 365 (16.0.13.127.21490), 32-bit
Office version: Office 365 (18.2008.12711.0)

**Comment by phw198** [comment]
Did `HKEY_CLASSES_ROOT\TypeLib\{00062FFF-0000-0000-C000-000000000046}` also have `9.4`? If it did, did you delete the `Win32\0` subfolder for it?

The rather more nuclear option is reinstalling Office - is that an option for you?

**Comment by heinoldenhuis** [comment]
I have the same issue on my machine since I upgrade from Windows 10 Pro 2004 to Windows 10 Pro 20H2.
After the upgrade the error occurred.
I have since reinstalled Office (Office16) and I only have the `9.6` key under `HKEY_CLASSES_ROOT\TypeLib\{00062FFF-0000-0000-C000-000000000046}`.

Tried to follow the steps and also register C:\Program Files (x86)\Microsoft Office\root\Office16\MSOUTL.OLB without any luck.
